src/GraphQL.Client.LocalExecution/GraphQLLocalExecutionClient.cs

```diff
@@ -95,16 +93,15 @@ private async Task<ExecutionResult> ExecuteAsync(GraphQLRequest request, Cancell
 
             var deserializedRequest = JsonConvert.DeserializeObject<GraphQLRequest>(serializedRequest);
             var inputs = deserializedRequest.Variables != null
```

Comment:
```suggestion
            var deserializedRequest = _documentSerializer.Deserialize<GraphQLRequest>(serializedRequest);
```
This is some of the dumbest code I've seen.  It could be greatly improved.

Reply:
Just to be clear, I'm not talking about your code, @sungam3r -- just mean the whole bit in general, and probably not so much the fault of the original programmer even but more due to how GraphQL.NET serialization has changed over the last few versions.  It just looks dumb now (lol).